src/utils/suggestions.js

```diff
@@ -93,7 +93,7 @@ const getSuggestions = (fields, suggestions, currentValue, suggestionProperties
 
 	suggestions.forEach((item) => {
 		fields.forEach((field) => {
-			parseField(item._source, field);
+			parseField({ ...item._source, _id: item._id }, field);
```

Comment:
We should also expose other meta fields that we normally expose in result components' hits. What do you think?

Reply:
We can definitely expose `_score`, but not able to think of a use case for `_index` & `_type`. Thoughts?

Reply:
Let's add it for the sake of consistency.